link/kprobe.go

```diff
@@ -141,7 +147,7 @@ func kprobe(symbol string, prog *ebpf.Program, opts *KprobeOptions, ret bool) (*
 	if prog == nil {
 		return nil, fmt.Errorf("prog cannot be nil: %w", errInvalidInput)
 	}
-	if !rgxTraceEvent.MatchString(symbol) {
+	if !rgxKprobe.MatchString(symbol) {
 		return nil, fmt.Errorf("symbol '%s' must be alphanumeric or underscore: %w", symbol, errInvalidInput)
```

Comment:
nit: should we change the error message to a more generic one?

Reply:
OK. Just updated.